service.ifttt/default.py

```diff
@@ -0,0 +1,112 @@
+import os, sys, re
+import xbmc, xbmcaddon, xbmcgui
+import json, urllib2
+
+__addon__ = xbmcaddon.Addon()
+__addonversion__ = __addon__.getAddonInfo('version')
+__addonid__ = __addon__.getAddonInfo('id')
+__addonname__ = __addon__.getAddonInfo('name')
+__addonPath__ = __addon__.getAddonInfo('path')
+__addonResourcePath__ = xbmc.translatePath(os.path.join(__addonPath__, 'resources', 'lib'))
+__addonIconFile__ = xbmc.translatePath(os.path.join(__addonPath__, 'icon.png'))
+sys.path.append(__addonResourcePath__)
```

Comment:
sorry please use absolute imports for your modules instead of using sys to append the paths.
Use something like `from resources.lib import my module`